mix.exs

```diff
@@ -0,0 +1,27 @@
+defmodule HookLineAndSlacker.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :hook_line_and_slacker,
+     version: "0.1.0",
+     elixir: "~> 1.4",
+     build_embedded: Mix.env == :prod,
+     start_permanent: Mix.env == :prod,
+     deps: deps()]
+  end
+
+  def application do
+    [extra_applications: [:logger],
+     mod: {HookLineAndSlacker, []}]
+  end
+
+  defp deps do
+    [
+      {:cowboy, "~> 1.1"},
+      {:plug, "~> 1.3"},
+      {:poison, "~> 3.0"},
+      {:slackex, "~> 0.0.1"},
```

Comment:
Is there a reason you chose to use `slackex`? Given the task is only creating one post request, does having a library like `slackex` make sense?